actions/restore.py

```diff
@@ -91,12 +92,12 @@ def unpack_resource():
 
 def is_v3_backup():
     ''' See if the backup file contains a db file indicating a v3 backup '''
-    cmd = "tar -tvf {0} --wildcards '*/db'".format(SNAPSHOT_ARCHIVE)
+    cmd = "tar -tvf {0} --wildcards '*/wal'".format(SNAPSHOT_ARCHIVE)
```

Comment:
I think the docstring of the function should be updated. Or else adding a comment would be helpful. Similar to what you mention in the PR description.

Reply:
Sure thing, missed that part. Will update it.